components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -1448,6 +1450,27 @@ func startContentInit(ctx context.Context, cfg *Config, wg *sync.WaitGroup, cst
 	cst.MarkContentReady(src)
 }
 
+func recordInitializerMetrics(path string, metrics *metrics.SupervisorMetrics) {
+	readyFile := filepath.Join(path, ".gitpod/ready")
```

Comment:
When using PVC approach, ready file is placed into /.workspace folder instead.
Example: https://github.com/gitpod-io/gitpod/blob/ff78e51f17bb9db9bf3a8fef603b6004cce3cf1d/components/supervisor/pkg/supervisor/supervisor.go#L1376

So we should probably do something similar here?

Reply:
My observation is that the ready file is still placed in /workspace/.gitpod/ready. I think the /.workspace ready file is placed there by ws-daemon but supervisor then writes its own ready file to /workspace. I have not changed anything regarding the location of the file, I am just piggybacking on it and adding additional content.